src/content/docs/mobile-monitoring/new-relic-mobile-ios/watchOS/watchOS-agent-setup.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Starting the New Relic agent to your watchOS project
+tags:
+  - Mobile monitoring
+  - New Relic Mobile iOS
+  - Installation
+  - SwiftUI
+  - WatchOS
+metaDescription: How to properly setup your watchOS project with the New Relic agent.
+freshnessValidatedDate: never
+---
+
+WatchOS projects do not normally contain an `ApplicationDelegate` or `ExtensionDelegate` class. These delegates are necessary to configure and start the New Relic agent in a watchOS project. Before adding the AppDelegate, follow the [outlined procedure](/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation) to correctly install the New Relic agent. 
+
+## WatchOS Application [#watchOS_App]
+
+To add an `AppDelegate` to your watchOS application, follow the steps:
+
+1. Add a new file to your project named `WatchAppDelegate.swift`
+1. Then in that file add a class named `WatchAppDelegate` that inherits from `NSObject` and conforms to the `WKApplicationDelegate` protocol. 
+1. In the `WatchAppDelegate` class you created, add the `applicationDidFinishLaunching` function. 
+1. As close to the start of `applicationDidFinishLaunching` as possible add `NewRelic.start(withApplicationToken: "APP_TOKEN")` replacing `APP_TOKEN` with your [application token](/docs/mobile-apps/viewing-your-application-token).
+
+<Callout variant="important">
+     To ensure proper instrumentation, you must start the agent on the first line of `applicationDidFinishLaunching()`, and run the agent on the main thread. Starting the call later, on a background thread, or asynchronously can cause unexpected or unstable behavior.
+ </Callout>
+    
+```swift
```

Comment:
! I see, is this an example of what a user _might_ get when they follow all of the previous steps?